File: python/1603_设计停车系统_easy.py
class ParkingSystem:

    def __init__(self, big: int, medium: int, small: int):
        self.data = [big, medium, small]

    def addCar(self, carType: int) -> bool:
        if self.data[carType-1]:
            self.data[carType-1] -= 1
        else:
            return False
        return True



# Your ParkingSystem object will be instantiated and called as such:
# obj = ParkingSystem(big, medium, small)
# param_1 = obj.addCar(carType)

File: cpp/1603_设计停车系统_easy.cpp
class ParkingSystem {
    int data[3];
public:
    ParkingSystem(int big, int medium, int small) {
        data[0] = big;
        data[1] = medium;
        data[2] = small;
    }
    
    bool addCar(int carType) {
        if (data[carType-1] > 0)
            data[carType-1]--;
        else
            return false;
        return true;   
    }
};

/**
 * Your ParkingSystem object will be instantiated and called as such:
 * ParkingSystem* obj = new ParkingSystem(big, medium, small);
 * bool param_1 = obj->addCar(carType);
 */